test/functional/rpc_net.py

```diff
@@ -133,6 +133,13 @@ def _test_getaddednodeinfo(self):
         assert_equal(added_nodes[0]['addednode'], ip_port)
         # check that a non-existent node returns an error
         assert_raises_rpc_error(-24, "Node has not been added", self.nodes[0].getaddednodeinfo, '1.1.1.1')
+        # check that node can not be added again
```

Comment:
Perhaps place this test (or all the new ones) two lines higher as it is order-dependent on the addnode call.

s/can not/cannot/